src/app/src/components/FacilityListItemsMatchTableRow.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { memo } from 'react';
+import { bool, func, number, oneOfType, shape, string } from 'prop-types';
+import TableRow from '@material-ui/core/TableRow';
+import TableCell from '@material-ui/core/TableCell';
+import isEqual from 'lodash/isEqual';
+
+import FacilityListItemsDetailedTableRowCell from './FacilityListItemsDetailedTableRowCell';
+
+import { listTableCellStyles } from '../util/styles';
+
+import { makeFacilityDetailLink } from '../util/util';
+
+import { facilityListItemStatusPropType } from '../util/propTypes';
+
+const propsAreEqual = (prevProps, nextProps) =>
+    isEqual(prevProps, nextProps)
+    && isEqual(prevProps.matchedFacility, nextProps.matchedFacility);
+
+const FacilityListItemsMatchTableRow = memo(({
+    rowIndex,
+    countryCode,
+    name,
+    address,
+    status,
+    hover,
+    matchedFacility,
+    handleSelectRow,
+}) => (
+    <TableRow
+        hover={hover}
+        onClick={handleSelectRow}
+    >
+        <TableCell
+            align="center"
+            padding="dense"
+            style={listTableCellStyles.rowIndexStyles}
+        >
+            <FacilityListItemsDetailedTableRowCell
+                title={rowIndex}
+                subtitle=" "
+                hrIsHidden
+                stringIsHidden
+                data={[matchedFacility.oar_id]}
+                hasActions={false}
+            />
+        </TableCell>
+        <TableCell
+            align="center"
+            padding="dense"
+            style={listTableCellStyles.countryCodeStyles}
+        >
+            <FacilityListItemsDetailedTableRowCell
+                title={countryCode || ' '}
+                subtitle=" "
+                hrIsHidden
+                stringIsHidden
+                data={[matchedFacility.oar_id]}
+                hasActions={false}
+            />
+        </TableCell>
+        <TableCell
+            padding="none"
+            style={listTableCellStyles.nameCellStyles}
+        >
+            <FacilityListItemsDetailedTableRowCell
+                title={name || ' '}
+                subtitle="Facility Match Name"
+                hsIsHidden={false}
```

Comment:
Typo? hs -> hr?

Reply:
Excellent catch! Will squash in a fixup to address.